### PR TITLE
MGMT-4079 Set CNV

### PR DESCRIPTION
**Pull request by pkliczewski**
In order to write end to end tests for cnv we need ability to enable cnv operator.

Signed-off-by: Piotr Kliczewski <piotr.kliczewski@gmail.com>

### CODE REVIEW

**Comment by pkliczewski** [comment]
@gamli75, @machacekondra, @masayag, @jordigilh, @jakub-dzon please review

**Comment by YuviGold** [comment]
This doesn't match the new API. 
Please take a look at the new [parameters](https://github.com/openshift/assisted-service/blob/master/swagger.yaml#L3091-L3099)

File: discovery-infra/test_infra/helper_classes/cluster.py [hunk 1]
```diff
@@ -133,6 +133,11 @@ def set_ocs(self, ocs_enabled):
         ocs_operator = {"operators": [{"operator_type": "ocs", "enabled": ocs_enabled}]}
         self.api_client.update_cluster(self.id, ocs_operator)
 
+    def set_cnv(self, cnv_enabled):
```
Comment (machacekondra) [inline discovery-infra/test_infra/helper_classes/cluster.py:136]: This is the new way: https://github.com/openshift/assisted-test-infra/pull/579/files

Comment (pkliczewski) [reply discovery-infra/test_infra/helper_classes/cluster.py:136]: Thanks, Will reuse

Comment (pkliczewski) [reply discovery-infra/test_infra/helper_classes/cluster.py:135]: @machacekondra rebased on top your changes and changed the code to reuse it for `CNV`

**Comment by pkliczewski** [comment]
@YuviGold please take a look

**Comment by pkliczewski** [comment]
@gamli75 please take a look

**Pull request merged**